### ISSUE TITLE
Allow using "Reparent to New Node" on the scene root

### ISSUE THREAD

**Issue by RedMser** [godotengine/godot-proposals#7241]
### Describe the project you are working on

UI-heavy applications

### Describe the problem or limitation you are having in your project

"Reparent to New Node" is a very useful tool in the editor for making UI (e.g. wrapping a Control into a new Container). But it does not work for the root node.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Show the menu option for "Reparent to New Node" on the scene root.

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Basically implement it like "Add Child Node" followed by "Make Scene Root".

### If this enhancement will not be used often, can it be worked around with a few lines of script?

It's editor usability.

### Is there a reason why this should be core and not an add-on in the asset library?

It's an improvement of an existing editor feature.

**Comment by cybernaut4** [comment]
I'm currently struggling with this. Even changing the Owner property doesn't make a difference.

Reparenting to the "main scene" while saving the value of who was its original owner while not meant to be instantiated by other types of parents as a separate scene allows flexibility on its own. At least for debugging purposes.
